src/main/java/calculator/CustomSeparator.java

```diff
@@ -0,0 +1,39 @@
+package calculator;
+
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+public class CustomSeparator {
+
+    private final String separator;
+
+    public CustomSeparator(final String separator) {
+        if(separator == null){
```

Comment:
자바 if 조건문 컨벤션은
```suggestion
        if (separator == null) {
```
입니다

Reply:
꼼꼼히 확인을 못했네요. 수정했습니다.